### PR TITLE
HDDS-3495. Clarify the document of OzoneFs

### PR DESCRIPTION
**Pull request by maobaolong**
## What changes were proposed in this pull request?

Notice that OzoneFileSystem and OzFs are a pair, BasicOzoneFileSystem and BasicOzFs is another pair. 

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-3495

## How was this patch tested?

Just document fix, no need to test.

### CODE REVIEW

**Comment by maobaolong** [comment]
@elek Please check if this test should be ignore, my PR contain only documents changes, no code at all.

image

**Comment by adoroszlai** [comment]
> Please check if this test should be ignore

It is now ignored in 51e9cfa52.

**Comment by maobaolong** [comment]
@adoroszlai Thank you, Please take a look at this PR, a put a important column in the table of the ozonefs document.

File: hadoop-hdds/docs/content/interface/OzoneFS.md [hunk 1]
```diff
@@ -145,11 +145,11 @@ This is the same implementation but doesn't include the features/dependencies wh
 
 The following table summarize which jar files and implementation should be used:
 
-Hadoop version | Required jar            | OzoneFileSystem implementation
----------------|-------------------------|----------------------------------------------------
-3.2            | filesystem-lib-current  | org.apache.hadoop.fs.ozone.OzoneFileSystem
-3.1            | filesystem-lib-legacy   | org.apache.hadoop.fs.ozone.OzoneFileSystem
-2.9            | filesystem-lib-legacy   | org.apache.hadoop.fs.ozone.BasicOzoneFileSystem
-2.7            | filesystem-lib-legacy   | org.apache.hadoop.fs.ozone.BasicOzoneFileSystem
+Hadoop version | Required jar            | OzoneFileSystem implementation | AbstractFileSystem implementation
```
Comment (adoroszlai) [inline hadoop-hdds/docs/content/interface/OzoneFS.md:148]:
```suggestion
Hadoop version | Required jar            | FileSystem implementation | AbstractFileSystem implementation
```

File: hadoop-hdds/docs/content/interface/OzoneFS.md [hunk 2]
```diff
@@ -145,11 +145,11 @@ This is the same implementation but doesn't include the features/dependencies wh
...
+---------------|-------------------------|-------------------------------------------------|-------------------------------------
+3.2            | filesystem-lib-current  | org.apache.hadoop.fs.ozone.OzoneFileSystem      | org.apache.hadoop.fs.ozone.OzFs
+3.1            | filesystem-lib-legacy   | org.apache.hadoop.fs.ozone.OzoneFileSystem      | org.apache.hadoop.fs.ozone.OzFs
+2.9            | filesystem-lib-legacy   | org.apache.hadoop.fs.ozone.BasicOzoneFileSystem | org.apache.hadoop.fs.ozone.BasicOzFs
+2.7            | filesystem-lib-legacy   | org.apache.hadoop.fs.ozone.BasicOzoneFileSystem | org.apache.hadoop.fs.ozone.BasicOzFs
  With this method the hadoop-ozone-filesystem-lib-legacy.jar can be used from
```
Comment (adoroszlai) [inline hadoop-hdds/docs/content/interface/OzoneFS.md:154]:
```suggestion

With this method the hadoop-ozone-filesystem-lib-legacy.jar can be used from
```

File: hadoop-hdds/docs/content/interface/OzoneFS.zh.md [hunk 3]
```diff
@@ -130,11 +130,11 @@ Ozone 文件系统的 jar 包有两种类型，它们都包含了所有的依赖
 
 下表总结了各个版本 Hadoop 应当使用的 jar 包和文件系统实现：
 
-Hadoop 版本 | 需要的 jar            | OzoneFileSystem 实现
----------------|-------------------------|----------------------------------------------------
-3.2            | filesystem-lib-current  | org.apache.hadoop.fs.ozone.OzoneFileSystem
-3.1            | filesystem-lib-legacy   | org.apache.hadoop.fs.ozone.OzoneFileSystem
-2.9            | filesystem-lib-legacy   | org.apache.hadoop.fs.ozone.BasicOzoneFileSystem
-2.7            | filesystem-lib-legacy   | org.apache.hadoop.fs.ozone.BasicOzoneFileSystem
+Hadoop 版本 | 需要的 jar            | OzoneFileSystem 实现  | AbstractFileSystem 实现
```
Comment (adoroszlai) [inline hadoop-hdds/docs/content/interface/OzoneFS.zh.md:133]:
```suggestion
Hadoop 版本 | 需要的 jar            | FileSystem 实现  | AbstractFileSystem 实现
```

**Comment by maobaolong** [comment]
@adoroszlai Thank you for your review and helping me to correct more fault. I have accept all of your suggestion and commit into this PR, PTAL.

**Pull request merged**